api/vpc/v1alpha2/vpc_types.go

```diff
@@ -329,7 +333,9 @@ func (vpc *VPC) Validate(ctx context.Context, kube client.Reader, fabricCfg *met
 			if !ipNet.Contains(ip) {
 				return nil, errors.Errorf("subnet %s: dhcp range start %s is not in the subnet", subnetName, subnetCfg.DHCP.Range.Start)
 			}
-
+			if ip := net.ParseIP(subnetCfg.DHCP.DNSServer); ip == nil && len(subnetCfg.DHCP.DNSServer) > 0 {
```

Comment:
Do we need to have a DNSServer IP from the VPC subnet or it could be any IP? If only from the subnet we should validate it

Reply:
nope it could be anything